File: Java/ROUND_2_A/src/test/java/FootballScoreStatsTests.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class FootballScoreStatsTests {

	@Test
	public void totalsFootballScoresForTeam() {
		FootballScoreStats stats = new FootballScoreStats();
		assertEquals(6, stats.teamTotal("Liverpool"));
	}

}

File: csharp/ROUND_2/ROUND_2_B/tests/FootballScoreStatsTests.cs
namespace ROUND_2_B.Tests;

public class FootballScoreStatsTests
{
    [Test]
    public void Test1()
    {
        FootballScoreStats stats = new FootballScoreStats(new FootballData());
        Assert.That(stats.TeamTotal("Liverpool"), Is.EqualTo(6));
    }
}